GitCommands/Git/GitRevision.cs

```diff
@@ -87,6 +87,7 @@ public BuildInfo BuildStatus
         [CanBeNull]
         public string Body { get; set; }
         public bool HasMultiLineMessage { get; set; }
+        public bool HasNotes { get; set; }
```

Comment:
This bool could be packed into a integer field along with `HasMultiLineMessage` to save space. Not a blocker for this PR though.